game_data.json

```diff
@@ -1,7 +1,22 @@
 {
```

Comment:
Can we delete this file now?

Reply:
This is still used locally... the game is still created on the json file and referenced for post-game adjustment... I didn't feel the need to put it into the database because it just gets overwritten every time